src/app/lib/utilities/preprocessor/rules/cpsAssetPage/convertToOptimoBlocks/blocks/image/index.test.js

```diff
@@ -0,0 +1,63 @@
+import clone from 'ramda/src/clone';
+import convertImage from '.';
+import {
+  cpsImageBlock,
+  captionBlock,
+  altTextBlock,
+  rawImageBlock,
+} from './fixture';
+
+describe('convertImage', () => {
+  it('should convert an image cps block into an optimo one', () => {
+    const expected = {
+      model: {
+        blocks: [captionBlock, altTextBlock, rawImageBlock],
+      },
+      type: 'image',
+    };
+
+    expect(convertImage(cpsImageBlock)).toEqual(expected);
+  });
+
+  it('should not include alt text block if alt text is missing', () => {
```

Comment:
Does missing `altText` block cause propType errors for the Image container? One to check and iterate on

Reply:
Yeah we need to decide where to put the business logic :) ile raise an issue